src/Tizen.Uix.InputMethod/Tizen.Uix.InputMethod/PredictionHintDataUpdatedEventArgs.cs

```diff
@@ -0,0 +1,44 @@
+﻿/*
+* Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+namespace Tizen.Uix.InputMethod
+{
+    /// <summary>
+    /// This class contains the data related to the PredictionHintDataUpdated event.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class PredictionHintDataUpdatedEventArgs
+    {
+        internal PredictionHintDataUpdatedEventArgs(string key, string keyValue)
+        {
+            Key = key;
+            KeyValue = keyValue;
+        }
+
+        /// <summary>
+        /// The prediction hint key.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Key { get; }
+
+        /// <summary>
+        /// The prediction hint value.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string KeyValue { get; }
```

Comment:
How about `Value`?
```suggestion
        public string Value{ get; }
```
This naming is used in KeyValuePair 
https://docs.microsoft.com/ko-kr/dotnet/api/system.collections.generic.keyvaluepair-2?view=netframework-4.7.2#%EC%86%8D%EC%84%B1

Reply:
Fixed.